backend/schema/schema_test.go

```diff
@@ -410,16 +410,16 @@ func TestParseEnum(t *testing.T) {
 	input := `
 module foo {
   // A comment
-  enum Color(String) {
-    Red("Red")
-    Blue("Blue")
-    Green("Green")
+  enum Color {
+	Red String = "Red"
```

Comment:
whoops looks like the spaces turned into tabs here

Reply:
ahh! i think that's ok actually, it looks like the other enum test strings have the same indent so i guess this is consistent 🤷‍♀️